workflow/tests/base_results/results.csv

```diff
@@ -310,8 +310,8 @@ base-pv-battery-outside-degrades.xml,34.98,8.43,14.53,0.0,0.0,0.0,0.0,0.0,0.0,0.
-base-schedules-detailed-stochastic-vacancy.xml,29.81,29.81,18.31,0.0,0.0,0.0,0.0,0.0,0.0,0.48,3.56,0.69,7.42,0.0,0.0,3.62,0.0,0.26,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.27,0.3,1.26,1.26,0.0,1.7,6.78,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.31,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,17.31,11.62,7.36,0.61,0.0,0.0,0.0,1.0,5370.0,5962.0,31.2,21.5,0.0,3.83,3.94,0.54,3.68,0.7,-0.25,0.0,0.0,2.64,-0.1,5.51,0.0,0.55,0.0,4.12,-7.93,0.0,-0.02,-0.45,-0.01,3.83,-0.02,8.9,0.0,0.0,-6.01,-0.06,-1.14,-6.54,-0.16,0.0,3.07,10.27,1141.0,884.0,9206.0,2113.0
-base-schedules-detailed-stochastic.xml,35.09,35.09,15.17,0.0,0.0,0.0,0.0,0.0,0.0,0.39,3.56,0.69,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.12,8.38,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.17,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.34,11.62,9.23,0.61,0.0,0.0,0.0,1.0,5370.0,5962.0,31.18,21.5,0.0,3.88,3.98,0.54,3.99,0.71,-0.13,0.0,0.0,3.06,-0.1,5.49,0.0,0.83,0.0,3.46,-11.47,0.0,-0.02,-0.45,-0.01,3.83,-0.02,8.9,0.0,0.0,-6.01,-0.06,-1.14,-6.54,-0.16,0.0,3.07,10.27,1355.0,998.0,11399.0,2616.0
+base-schedules-detailed-stochastic-vacancy.xml,32.37,32.37,16.83,0.0,0.0,0.0,0.0,0.0,0.0,0.44,3.55,0.69,8.31,0.0,0.0,4.04,0.0,0.3,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.29,0.33,1.38,1.37,0.0,1.9,7.55,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,16.83,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.91,11.6,8.32,0.61,0.0,0.0,0.0,1.0,4702.0,5523.0,31.07,21.49,0.0,3.85,3.96,0.54,3.82,0.7,-0.19,0.0,0.0,2.83,-0.1,5.49,0.0,0.7,0.0,3.81,-9.61,0.0,-0.02,-0.45,-0.01,3.83,-0.03,8.9,0.0,0.0,-6.01,-0.05,-1.14,-6.55,-0.16,0.0,3.07,10.27,1216.0,936.0,10372.0,2380.0
+base-schedules-detailed-stochastic.xml,35.08,35.08,15.16,0.0,0.0,0.0,0.0,0.0,0.0,0.39,3.55,0.69,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.12,8.38,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.16,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.33,11.6,9.23,0.61,0.0,0.0,0.0,1.0,4702.0,5523.0,31.07,21.49,0.0,3.88,3.98,0.54,3.99,0.71,-0.13,0.0,0.0,3.06,-0.1,5.49,0.0,0.83,0.0,3.46,-11.47,0.0,-0.02,-0.45,-0.01,3.83,-0.03,8.9,0.0,0.0,-6.01,-0.05,-1.14,-6.55,-0.16,0.0,3.07,10.27,1355.0,998.0,11397.0,2615.0
```

Comment:
Why did you have to make changes to these schedule files?

Reply:
Well, we're calling BuildResidentialScheduleFile now to generate those files. Apparently they got updated. It appears the vacancy period is something other than Dec 15 - Jan 15 -- I'll look into `stochastic-vacancy.csv`. Not sure about `stochastic.csv`.

Reply:
Ah, right. Awesome! I hadn't even thought of the benefit that we'll now be ensuring that those files stay up to date (and seeing if they ever accidentally/incorrectly change).

Reply:
I'd still like to understand why the schedules have changes. In particular, the `hot_water_foo` columns in stochastic.csv are quite a bit different than before. Can we figure out why?

Reply:
One thing I think of is the pseudo random number generator `prng` that floats around in `schedules.rb`. The order that it's used determines random offsets.

This is hard to track down.

Reply:
What do you mean by "the order that it's used"?

Reply:
For example, if in this example `sinks` came before `showers`, you'd get something different: https://github.com/NREL/OpenStudio-HPXML/blob/master/BuildResidentialScheduleFile/resources/schedules.rb#L711-L723

Reply:
I see. Okay, I won't belabor the point any further.

Reply:
But I do think those schedule csv files will be much more tractable. Starting now...